components/textfield/stories/textfield.stories.js

```diff
@@ -80,16 +76,36 @@ export default {
 		isRequired,
 		isReadOnly,
 		isLoading,
-		pattern: {
-			name: "Pattern",
+		pattern: { table: { disable: true } },
```

Comment:
What is the reasoning to removing this from the control table? I don't quite understand `pattern`, but I do see that [Rise still has it in S2](https://github.com/adobe/spectrum-css/blob/f88692a79adc681c8ed26a633fef6ad4ba76663d/components/textfield/stories/textfield.stories.js#L175)